Review request for Set up a "Hello Microverse" project

ShaliniNayan opened the pull request:
Create new feature branches.
Followed GitHub flow.
Open pull requests in GitHub.
Write descriptive, easy to understand commit messages.
Write short, detailed, easy to understand descriptions of pull requests.
Write a descriptive and easy to understand README in business English for every project submitted.
Demonstrated an ability to submit a project for code review.

ShaliniNayan committed:
Added newbranch, index.html.,style.css & modified

ShaliniNayan committed:
Modified 2 space in style.css

ShaliniNayan committed:
Added test.md

ShaliniNayan committed:
Add class to header

souad988 commented on test.md:
```diff
@@ -0,0 +1 @@
+"This file should be ignored by git".
```
[OPTIONAL] Well done on adding this file to gitignore  but since ignored it should not appear in github you can use this command to remove it from git cache $ git rm –cached <filename>” command. and always add files you want to ignore to gitignore before you add them to your code base

souad988 reviewed (changes requested):
Hi @ShaliniNayan , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

![212385281-4797c58a-d358-423a-aa7c-832c22203865](https://user-images.githubusercontent.com/59707859/227177142-c79a7f72-f319-4f56-8e52-eb576f80ef01.gif)

 ## Highlights
✔Use Microverse README template 
✔Good efforts

 ## Required Changes ♻️
_Well done by implementing some of requested changes by the previous reviewer but you README still needs improvements before mentioning changes please always do all the work in the feature branch you will submit for review as here I don't have access to your readme so I can't give inline feedback  , First thing is the title and logo use title that describes your project and if you don't have logo just remove it , next is the default text under each section it's important to remove it after customizing with your own content , and finally the link to this project licence file is broken it's important for legal reasons to add a valid licence file to your repo "you can use the template referd to in that section" and put a valid link to it in the README  _
_One more thing was requested before if your PR title and description those are meat to give a clear idea about the work done in the feature exp: setup linter / hello microverse
 ...
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ShaliniNayan committed:
Update README.md

Update README.md

ShaliniNayan committed:
Create licence

Update licence.

ShaliniNayan committed:
Update README.md

Update licence

ShaliniNayan committed:
Create license

Added new license file.

ShaliniNayan committed:
Add linters

ShaliniNayan committed:
Update README.md

Updated README,md file.

ShaliniNayan commented:
Created new feature branch as requested.
New feature branch name is First_Project.
Updated my README to look more professional.
Linked License to README file as requested.
Updated my PR entitle to more professional.

abdulhamiid reviewed (changes requested):
# Status: Required Changes :recycle:
Hi @ShaliniNayan 👋, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, I really appreciate the effort you put in! but there are still some which haven't been addressed yet.

![almost-there](https://user-images.githubusercontent.com/57562869/143511358-9cde7daf-4ad0-4db2-8f15-18b54288ff91.gif)

## Highlights 🌟

- ✔️ Descriptions are very descriptive
- ✔️ Use GitHub Flow
- ✔️ Meaningful commit messages.

## Required Changes ♻️
_Please kindly check the comments under the review._

### 💡 Linters Setup

- [ ] According to the project requirements, we are required to set up linters correctly and make sure that there are [no linter errors](https://github.com/microverseinc/linters-config).

![image](https://user-images.githubusercontent.com/78251927/227289071-c1363182-08f0-4002-810f-c19e5518eb07.png)

- [ ] Currently, your last commit does not have a green check mark ✔️ which means that there is a problem with the linters setup. please let's fix that 🙏

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ShaliniNayan committed:
Delete license

ShaliniNayan committed:
Update README.md

ShaliniNayan committed:
update lighthouse

ShaliniNayan committed:
Merge branch 'First_Project' of github.com:ShaliniNayan/Hello-world into First_Project

ShaliniNayan commented:
Fixed Linters.

christianbiring1 reviewed (approved):
Hi @ShaliniNayan , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 
# STATUS: APPROVED :tada: 
![imageswelljob](https://user-images.githubusercontent.com/97100091/227308664-afcb23fb-c7f0-4bde-850c-d3e1c6171908.jpeg)

## Highlights
- [ ] Descriptive PR `title` and `summary` :+1: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ShaliniNayan committed:
Create LICENSE.md
